# Software

Currently, a pletora of programs are available for analysis of proteomics data analysis. However, MaxQuant is one of the most popular because it is easy to use, offers functionalities for nearly all kinds of proteomics data analysis challenges and it's free. The software is maintaned by the Max Planck Institute of Biochemistry (MPI) and receives updates regularly. 

You can download **MaxQuant** [here](https://www.maxquant.org/maxquant/)

> Cox, J., Mann, M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat Biotechnol 26, 1367–1372 (2008). https://doi.org/10.1038/nbt.1511

Additionally, the Perseus program handles most output files from MaxQuant and assists the analysis.

You can download **Perseus** [here](https://www.maxquant.org/perseus/)

> Tyanova, S., Temu, T., Sinitcyn, P. et al. The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat Methods 13, 731–740 (2016). https://doi.org/10.1038/nmeth.3901

Both programs with their graphical user interfaces (GUI) are only currently available for Windows Operating Systems, althougth MaxQuant can run in UNIX enviroment with assistance of Mono/DotNet. 

------

## Additional tools and databases
<br />
### AlphaMap (**GUI needs to be installed**):
<br />
Link: [https://mannlabs.github.io/alphamap/#gui](https://mannlabs.github.io/alphamap/#gui) (GUI)

> Voytik, E., Bludau, I., Willems, S. et al. AlphaMap: an open-source Python package for the visual annotation of proteomics data with sequence-specific knowledge. 2022. Bioinformatics.

### PLAZA (**Please make an account beforehand**)
<br />
Link: [https://bioinformatics.psb.ugent.be/plaza/](https://bioinformatics.psb.ugent.be/plaza/)
<br />
Registration: [https://bioinformatics.psb.ugent.be/plaza/versions/plaza_v5_dicots/workbench/logon](https://bioinformatics.psb.ugent.be/plaza/versions/plaza_v5_dicots/workbench/logon)
<br />
Choose “PLAZA” under “Authentication type” and click “Register” to sign up for a new account
> Van Bel, M., Silvestri, F., Weitz, E. M., Kreft, L., Botzki, A., Coppens, F., Vandepoele, K. (2022) PLAZA 5.0: extending the scope and power of comparative and functional genomics in plants. 2022. Nucleic Acids Res.

### PhosphositePlus
<br />
Link: [https://www.phosphosite.org/homeAction.action](https://www.phosphosite.org/homeAction.action
)
> Hornbeck PV, Chabra I, Kornhauser JM, Skrzypek E, Zhang B. PhosphoSite: A bioinformatics resource dedicated to physiological protein phosphorylation. 2004. Proteomics.

### AlphaFold
<br />
Link: [https://alphafold.ebi.ac.uk/](https://alphafold.ebi.ac.uk/)

> Jumper, J., Evans, R., Pritzel, A. et al. Highly accurate protein structure prediction with AlphaFold. 2021. Nature.

### Plant PTM Viewer
<br />
Link: [https://www.psb.ugent.be/webtools/ptm-viewer/](https://www.psb.ugent.be/webtools/ptm-viewer/)

> Willems P, Horne A, Van Parys T et al. The Plant PTM Viewer, a central resource for exploring plant protein modifications. 2019. Plant J.

### PhosPhAt
<br />
Link: [https://phosphat.uni-hohenheim.de/phosphat.html](https://phosphat.uni-hohenheim.de/phosphat.html)

> Heazlewood JL, Durek P, Hummel J, Selbig J, Weckwerth W, Walther D, Schulze WX. PhosPhAt: A Database of phosphorylation sites in Arabidopsis thaliana and a plant specific phosphorylation site predictor. 2008. Nucleic Acids Research.

